presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/TestSwapAdjacentWindowsByPartitionsOrder.java

```diff
@@ -104,4 +105,26 @@ public void subsetComesFirst()
                                 ImmutableList.of(),
                                 values(ImmutableMap.of("a", 0, "b", 1)))));
     }
+
+    @Test
+    public void dependentWindowsAreNotReordered()
```

Comment:
please squash this commit, and probably my previous commit (with missing test) is solved with that